docs/source/docs/pipelines/about-pipelines.md

```diff
@@ -44,3 +56,9 @@ AprilTag / AruCo Pipelines have 3 steps:
 1. Input: This is the same as the above.
 2. AprilTag: This step include AprilTag specific tuning parameters, such as decimate, blur, threads, pose iterations, and more.
 3. Output: This is the same as the above.
+
+Object Detection Pipelines have 3 steps:
+
+1. Input: This is the same as the above.
```

Comment:
maybe just refactor this section so we don't just have "input/output" this is the same as above repeated but its fine either way